ranking-service/src/main/resources/application-test.yml

```
@@ -0,0 +1,15 @@
+  config:
+    import:
+      - optional:file:../.env[.properties]
+      - classpath:./properties/datasource.yml
+      - classpath:./properties/jpa.yml
+
+eureka:
+  client:
+    service-url:
+      defaultZone: ${EUREKA_SERVER}
+server:
+  port: ${RANKING_SERVER}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)